collect_app/src/main/res/layout/project_settings_dialog_layout.xml

```diff
@@ -53,21 +53,20 @@
         android:id="@+id/general_settings_button"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        style="@style/Widget.MaterialComponents.Button.OutlinedButton"
+        style="@style/Widget.Collect.Button.OutlinedButton"
         android:layout_marginTop="@dimen/margin_small"
         android:text="@string/general_preferences"
-        android:textColor="@color/deep_purple_500"
+        android:textColor="?colorOnPrimary"
```

Comment:
This should be `colorOnSurface` instead of `colorPrimary`. In our light and dark themes they happen to match, but you'll see this doesn't work in Magenta where they are different. There's some okish docs on the colors and what they're used for [here](https://material.io/develop/android/theming/color). I'm also thinking this could just go in the style?

Reply:
Right, fixed.